GOVERNANCE.md

```diff
@@ -0,0 +1,100 @@
+# Appium Project Governance
+
+The Appium Project wants as much as possible to operate using procedures that
+are fair, open, inviting, and ultimately good for the community. For that
+reason we find it valuable to codify some of the ways that the Project goes
+about its day-to-day business. We want to make sure that no matter who you are,
+you have the opportunity to contribute to Appium. We want to make sure that no
+one corporation can exert undue influence on the community or hold the Project
+hostage. And likewise we want to make sure that corporations which benefit from
+Appium are also incentivized to give back.
+
+### The Technical Committee
+
+The project is officially led by a Technical Committee (TC), currently
+consisting of:
+
+* [@jlipps](https://github.com/jlipps)
```

Comment:
@boneskull you should also be in this list.
Also, are we going to copy this document to all monoreps? I assume the governance is the same for the whole Appium project, not per repository

Reply:
indeed. maybe we should just link.